### ISSUE TITLE
SpaceFM Support

### ISSUE THREAD

**Issue by 0pLuS0** [cbrnix/Newaita#2]
Hi,

I really like the look of what I am seeing so far.

Menu & Bookmark icons don't appear properly, there might be more, I only had a brief look, but can you please get the icon set to properly support SpaceFM? 

https://github.com/IgnorantGuru/spacefm

I'm also only running Openbox, so I just place the directory in /home/foo/.icons

So maybe the icon set also needs better support for people running small window managers, who only manually add the icons in by hand...

Thanks

**Comment by cbrnix** [comment]
Can you attach a screenshot of the problem? Unfortunately, I do not know the structure of this FM. I'll try to think of something

**Comment by 0pLuS0** [comment]
Hi,

There's nothing unusual about SpaceFM, it just appears that you are missing quite a few of the standard Gnome/GTK icons is all...

Like the icons for ```Actions Places Status```

I'm attaching a few screen shots to show a little of what's missing for me.

This screen shot, ```Panel 3 View``` & ```Preferences``` this is using a Stock Gnome/GTK icon, this is one you are missing.

![spacefm_1](https://user-images.githubusercontent.com/36190223/42916807-7dd3e26a-8af6-11e8-9500-57ef39d0c9f1.jpg)


This screen shot, the bookmark icons are missing

![spacefm_2](https://user-images.githubusercontent.com/36190223/42916832-9db23ad2-8af6-11e8-9c10-2c671d689eb1.jpg)


Since SpaceFM is multi pane it uses an icon at the bottom of each pane that is showing you which pane you are on. When I use the Faenza iconset, it appears to be using;

symlinked stock_yes.png -> ./gtk-yes.png

There's also a Permissions icon missing, or the Name Permissions is using what appears in Faenza as;

symlinked  gtk-dialog-authentication.png -> ./dialog-password.png

This Permissions section appears as a lock, from the standard Gnome/GTK icons.

**Comment by cbrnix** [comment]
try now, please :)

**Comment by 0pLuS0** [comment]
@cbrnix getting there, now I see preferences icons, and the panel checkmark icons, but the folder/bookmarks still not appearing.

By the way, the checkmarks are not as dark black as I'm seeing other icons done in black, these to be a little darker black.

**Comment by cbrnix** [comment]
Bookmark folders I added. As for the shades, this is strange, because the color is the same for all actions

**Comment by 0pLuS0** [comment]
@cbrnix getting there, now I see preferences icons, and the panel checkmark icons, but the folder/bookmarks still not appearing.

By the way, the checkmarks are not as dark black as I'm seeing other icons done in black, these to be a little darker black.

I also see a few other icons missing, it would be easier to install SpaceFM on your box, it's extremely small and light weight, who knows, you may even like it... hehe 

But in the meantime, here are some more screen shots.

Given how the iconset's overall look is black and white abstract icons, I'd stick to this look for everything, it looks nice.

This image, the yellow caution, I'd do it all in black and white, and there's a red dot, which in another screen shot I'll show, appears as a red X, so for now, I'm assuming this red X is part of your iconset?


![delete](https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg)


Here's the red X, is that your icon? If so, the red dot above needs to appear the same...


![rename](https://user-images.githubusercontent.com/36190223/43028613-69f5864c-8c70-11e8-9a61-7844a4d80b7c.jpg)


This Path Bar icon looks out of place, notice the thin line look of the icon Delete? I'd make this one also in a thin red look too...


![pathbar](https://user-images.githubusercontent.com/36190223/43028656-99c2180e-8c70-11e8-9af3-e9559df8dbb5.jpg)

**Comment by 0pLuS0** [comment]
@cbrnix getting there, now I see preferences icons, and the panel checkmark icons, but the folder/bookmarks still not appearing.

By the way, the checkmarks are not as dark black as I'm seeing other icons done in black, these to be a little darker black.

I also see a few other icons missing, it would be easier to install SpaceFM on your box, it's extremely small and light weight, who knows, you may even like it... hehe 

But in the meantime, here are some more screen shots.

Given how the iconset's overall look is black and white abstract icons, I'd stick to this look for everything, it looks nice.

```This image, the yellow caution, I'd do it all in black outline, how most of the black icons already appear, and there's a red dot, which in another screen shot I'll show, appears as a red X, so for now, I'm assuming this red X is part of your iconset?```


![delete](https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg)


```Here's the red X, is that your icon? If so, the red dot above needs to appear the same...```


![rename](https://user-images.githubusercontent.com/36190223/43028613-69f5864c-8c70-11e8-9a61-7844a4d80b7c.jpg)


```This Path Bar icon looks out of place, notice the thin line look of the icon Delete to the left? I'd make this Path Bar one also in a thin red look too...```


![pathbar](https://user-images.githubusercontent.com/36190223/43028656-99c2180e-8c70-11e8-9af3-e9559df8dbb5.jpg)

**Comment by cbrnix** [comment]
Added gtk-dir folder now. 
Cancel, path bar, No - it's not my icons. Need more links, now i see

**Comment by cbrnix** [comment]
Cancel links added. Perhaps now it will be normal. I did not understand how to define the icon path bar

**Comment by 0pLuS0** [comment]
@cbrnix getting there, now I see preferences icons, and the panel checkmark icons, but the folder/bookmarks still not appearing.

By the way, the checkmarks are not as dark black as I'm seeing other icons done in black, these need to be a little darker black.

I also see a few other icons missing, it would be easier to install SpaceFM on your box, it's extremely small and light weight, who knows, you may even like it... hehe 

But in the meantime, here are some more screen shots.

Given how the iconset's overall look is black and white abstract icons, I'd stick to this look for everything, it looks nice.

```This image, the yellow caution, I'd do it all in black outline, how most of the black icons already appear, and there's a red dot, which in another screen shot I'll show, appears as a red X, so for now, I'm assuming this red X is part of your iconset?```


![delete](https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg)


```Here's the red X, is that your icon? If so, the red dot above needs to appear the same...```


![rename](https://user-images.githubusercontent.com/36190223/43028613-69f5864c-8c70-11e8-9a61-7844a4d80b7c.jpg)


```This Path Bar icon looks out of place, notice the thin line look of the icon Delete to the left? I'd make this Path Bar one also in a thin red look too... Also notice the File List, and Devices, I'd change these to a black outline look too.```


![pathbar](https://user-images.githubusercontent.com/36190223/43028656-99c2180e-8c70-11e8-9af3-e9559df8dbb5.jpg)


Remember, the majority of icons that I've noticed are in a black outline, that is the look you want to stick to for all your icons. I mean there are some icons in some color, but they are all this flat abstract look. So then having the devices and file list icon, is not the same style... 

Here's the [b]Contrast[/b] I am talking about... The ```Actions``` icon look is not the same as the ```Apps```, of course you are the creator, but I think this is to much contrast, I personally love the look of the ```Actions``` icons and would stick to this sort of look for the overall iconset...

But for some of the files and mimes, maybe a little color I see is nice to have, for a nice contrast, but then these icons I think should also be in the Black Outline look;

application-vnd.nintendo.snes.rom.svg
application-x-cd-image.svg
application-x-clementine.svg
application-x-kdenlivetitle.svg

I'm a very Artsy person, and I truly believe in a Balance to the look, especially with something like this, but when you look at it, it's like you have several concepts going on, confused as to what the overall THEME look is suppose to be... Make sense?

Anyhow, I hope you appreciate my feedback...

The ```Actions``` is a perfect example of how you created this look, but didn't extend it to other icons, like the application-x-cd-image.svg, makes no sense, when you already have a look of cds in the Actions, in black outline...

And in these Actions icons, you splash a little color in there, that too should be the overall theme idea, spread through out all the other icons, black outlined, in some colors here and there, and then when you want something like a color, the call-start call-stop are great examples of color, that match the overall feel/look of the theme...

So in a nutshell these are all out of place;  ```'Apps' 'categories' 'devices' - panel needs for of the outline look to all the icons that aren't but overall it has the look better... Places, I'd keep only the folders and trash cans, the others in the outline look, or color like call-start call-stop look. Status, same like panel, needs more outline, or colored look like call-start call-stop...

Anyhow I hope you get the idea, the iconset basically looks like 2 going on at the same time, thrown together from two different projects.

Keep up the great work!

**Comment by 0pLuS0** [comment]
@cbrnix sorry I was editing over my reply, when you replied, please go back and read everything below the last image...

Thanks

**Comment by cbrnix** [comment]
The fact is that at the moment the action icons are a temporary measure to build up the structure to the end. I already have sketches (by the way, they have rounded corners, which is combined with the general style). I make icons alone, out of duty, and so it's hard for me to do everything at once :) While I'm focused on applications. So sorry that the delay or not immediately comes out expected.

**Comment by 0pLuS0** [comment]
No need to aplogize about delays, I'm just here trying to help and share is all...

You're doing great, I can't wait to see when it's completed.

Keep up the great work!

**Comment by 0pLuS0** [comment]
@cbrnix 

No need to aplogize about delays, I'm just here trying to help and share is all...

You're doing great, I can't wait to see when it's completed.

Keep up the great work!

Also, I noticed you have different color folders, I can't remember how with SpaceFM I could use different color folders?

Also could you add in a light blue shade like the look I'm attaching?

![folder](https://user-images.githubusercontent.com/36190223/43031376-1f5c14fa-8c90-11e8-8479-e91341fd0949.jpg)

**Comment by 0pLuS0** [comment]
If you have Gimp installed, I see icons missing in the menus for gimp, if you look at them all, like;

```File Edit Select etc...```

Also if you have Geany, it's missing icons.


This screen shot shows it using some Gnome icon for the Check mark, and you have a thin X, which I understand is yours

![buttons](https://user-images.githubusercontent.com/36190223/43106260-77683852-8ec7-11e8-835f-944b0fcff09f.jpg)


In this screen shot I am using for Yad, which is like Zenity, to create bash GUI, it's missing icons for the X, OK, and Logout.

These are the icons I used in it; ```window-icon=gnome-shutdown - image=gnome-shutdown - button="gtk-close:1" - button="gtk-ok:0"```


![yad](https://user-images.githubusercontent.com/36190223/43106303-9c93a8aa-8ec7-11e8-875e-a84f47f721e5.jpg)

**Comment by cbrnix** [comment]
As far as I understand, the gimp uses its own set of icons, is not it? 
The rest gradually I decide. Not all at once :) Therefore, until now Alpha

**Comment by 0pLuS0** [comment]
Sorry if I'm jumping fast here, I know you are working on the iconset, I just thought I'd show you what I see is missing is all, to help.

Every icon set I've used so far has replacement icons for these;

gnome-shutdown - I don't see an icon called this, I see system-shutdown, and dialog-question being used from some iconsets....
gtk-close
gtk-ok

Sorry again if I'm jumping here...

Thanks

**Comment by 0pLuS0** [comment]
My bad on gimp, it looks like in the menus it will use a little from an iconset and mostly it's own.

Sorry if I'm jumping fast here, I know you are working on the iconset, I just thought I'd show you what I see is missing is all, to help.

Every icon set I've used so far has replacement icons for these;

```gnome-shutdown``` - I see ```system-shutdown``` and dialog-question being used for this name...
```gtk-close```
```gtk-ok```

Sorry again if I'm jumping here...

Thanks

**Comment by cbrnix** [comment]
You are very helpful in this. Thank you! I just sometimes did not have time to finish something yet. Simlink also require a little time :)

**Comment by 0pLuS0** [comment]
I hear ya, lots of work, keep up the great work, it's looking nice! :+1:

**Comment by cbrnix** [comment]
While I can not add symlinks, since I completely re-arrange the actions to a new style
![screenshot_20180724_024525](https://user-images.githubusercontent.com/40643537/43108742-019eca26-8eec-11e8-96d3-dea95f7ecb29.png)

**Comment by 0pLuS0** [comment]
Sorry, didn't understand your last reply, what does this mean?

```While I can not add symlinks, since I completely re-arrange the actions to a new style``` ?

**Comment by cbrnix** [comment]
I mean, it's not temporary that the actions go around like they are now

**Comment by 0pLuS0** [comment]
Don't you mean it's not permanent, but the work you are doing on the icons is temporary? :yum:

**Comment by 0pLuS0** [comment]
Hi,

Not sure where you at, but I thought I'd give you a heads up for what I'm still seeing.

Caution icon and red dot, both should be changed;

![delete](https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg)

Computer Device icon, should have an ```Outline``` design like most appear in UI Menus. Also now ```Blue Plus``` sign icons appear for devices, this wasn't happening before;

![devices_icon](https://user-images.githubusercontent.com/36190223/43671183-edeaa74c-9784-11e8-972c-c27608e2a7ab.jpg)


SpaceFM Home icons that are used for ```Homepage SpaceFM News Get Plugins``` not sure if this is your icon now, but a ```Outline``` look for a House icon would be better

![home_icon](https://user-images.githubusercontent.com/36190223/43671192-1436331c-9785-11e8-8689-22d88afa0075.jpg)

This house should be used for the Menu above;

![house](https://user-images.githubusercontent.com/36190223/43671209-60cfc1ca-9785-11e8-8925-2a52d5be095f.jpg)

Info icon needs replacing, again, a nice ```Outline``` look;

![info](https://user-images.githubusercontent.com/36190223/43671211-69cb763e-9785-11e8-858f-67f4bf854e06.jpg)

Open URL, is this your world icon? Need a nice ```Outline``` look for the world;

![menu](https://user-images.githubusercontent.com/36190223/43671242-d6d00650-9785-11e8-96fe-fe1a57cca360.jpg)

Ok button needs a replaced, ```Outlined``` arrow;

![ok](https://user-images.githubusercontent.com/36190223/43671225-8fcb396e-9785-11e8-9ab4-879464bc6c09.jpg)

**Comment by cbrnix** [comment]
I added some links to the dialog icons. Homepage, SpaceFM, News and Get Plugins are both an application icon and making it to others does not make much sense...

**Comment by 0pLuS0** [comment]
News and Get Plugins, making it to others, not sure I understand?

I just assumed you use the House icon you created for these? No extra work needed...

Hmm

**Comment by cbrnix** [comment]
Spacefm takes these icons from apps. It is called spacefm.svg and is also an icon for spacefm itself













f I change it, then the icon in the main menu will be outline.

**Comment by 0pLuS0** [comment]
News & Get Plugins... ok
Upd?

**Comment by 0pLuS0** [comment]
@cbrnix why did you close this?

There are still issues, I just cloned today your latest work and I still see some of these same problems I submitted.

Bookmark icons still missing;
https://user-images.githubusercontent.com/36190223/42916832-9db23ad2-8af6-11e8-9c10-2c671d689eb1.jpg

OK button still shows this blue arrrow;
https://user-images.githubusercontent.com/36190223/43106303-9c93a8aa-8ec7-11e8-875e-a84f47f721e5.jpg

Yellow Warning Triangle and NO Button still the same;
https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg

Open URL shows a picture of the world
https://user-images.githubusercontent.com/36190223/43671242-d6d00650-9785-11e8-96fe-fe1a57cca360.jpg

And these additional screen shots too...

Font Icon?
![font](https://user-images.githubusercontent.com/36190223/46512414-6523a900-c843-11e8-8402-d5a89c9f946f.jpg)

Open URL?
![open_url](https://user-images.githubusercontent.com/36190223/46512421-6e147a80-c843-11e8-8407-3ba15cd4c4fe.jpg)

Save Button - 2 Blue Floppies
![save_geany](https://user-images.githubusercontent.com/36190223/46512426-753b8880-c843-11e8-9c6e-35ea180bae5b.jpg)

THANKS

**Comment by 0pLuS0** [comment]
@cbrnix why did you close this?

There are still issues, I just cloned today your latest work and I still see some of these same problems I submitted.

Bookmark icons still missing;
https://user-images.githubusercontent.com/36190223/42916832-9db23ad2-8af6-11e8-9c10-2c671d689eb1.jpg

OK button still shows this blue arrrow;
https://user-images.githubusercontent.com/36190223/43106303-9c93a8aa-8ec7-11e8-875e-a84f47f721e5.jpg

Yellow Warning Triangle and NO Button still the same;
https://user-images.githubusercontent.com/36190223/43028580-2fde48c2-8c70-11e8-8f93-4379b0a5de8b.jpg


And these additional screen shots too...

Font Icon?
![font](https://user-images.githubusercontent.com/36190223/46512414-6523a900-c843-11e8-8402-d5a89c9f946f.jpg)

Open URL?
![open_url](https://user-images.githubusercontent.com/36190223/46512421-6e147a80-c843-11e8-8407-3ba15cd4c4fe.jpg)

Save Button - 2 Blue Floppies - ```This appears in Geany IDE```
![save_geany](https://user-images.githubusercontent.com/36190223/46512426-753b8880-c843-11e8-9c6e-35ea180bae5b.jpg)

THANKS

**Comment by cbrnix** [comment]
Can you take a screenshot with another theme (for example, papirus)?

**Comment by 0pLuS0** [comment]
The icon set?
https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/releases

If so here ya go! 

![papirus](https://user-images.githubusercontent.com/36190223/46519664-b4300500-c868-11e8-9bf3-61a12384d76c.jpg)

**Comment by cbrnix** [comment]
I mean dialog window

**Comment by cbrnix** [comment]
Can you post a screenshot of dialog boxes with a different theme (delete item, etc.)? It would be easier for me to understand which icons I missed ....

**Comment by 0pLuS0** [comment]
I highly suggest just installing SpaceFM, it's small/lightweight, then it will be easier for you to support in the future.

I'd assume you're on a typical Distro that has it in the Package Management for easy install...

**Comment by cbrnix** [comment]
It is installed. However, it uses all the dialogs from DM, so I don’t see those icons that are required

**Comment by 0pLuS0** [comment]
GTK theme or iconset?

**Comment by cbrnix** [comment]
> GTK theme or iconset?
just make a screenshot of the no-yes dialogue with the papirus pls
